src/components/learning-guide-chart.coffee

```diff
@@ -1,19 +1,21 @@
 _ = require 'underscore'
 d3 = require 'd3'
 
-CLOUDS_PATH = '/style/resources/clouds.svg'
-PLANE_PATH = '/style/resources/openstax-plane.svg'
-CITYSCAPE_PATH = '/style/resources/cityscape.svg'
+CLOUD_PATH = '/style/resources/learning-guide/cloud.svg'
```

Comment:
Also, these assets will probably need to be included via CSS files to load properly (using `background-image: `) because these assets do not load when loaded from `:3001`. I had to change each path to `CLOUD_PATH = '//localhost:8000/dist/cloud.svg'` to get it to work.